src/openyieldtables/data/yield_tables_meta.csv

```diff
@@ -86,3 +86,4 @@ id;title;country_codes;type;source;link;yield_class_step;age_step;tree_type
 180;Spruce Hungary Taeg Sopron;HU;dgz_103;Taeg Sopron;;1;10;coniferous
 181;Pine Hungary Taeg Sopron;HU;dgz_104;Taeg Sopron;;1;10;coniferous
 182;Robinia Hungary Fatermési Táblák;HU;dgz_100;Taeg Sopron Fatermési Táblák;;1;5;deciduous
+183;Stieleiche Hungary Fatermési Táblák;HU;dgz_101;Taeg Sopron Fatermési Táblák;;1;5;deciduous

```

Comment:
dgz_101 - is this correct?

Reply:
yes, the type for each tree species was provided by @Erikkue